src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -513,7 +510,7 @@ public static ConstantValue GetDefaultValue(this TypeSymbol type)
             {
                 if (type.IsEnumType())
                 {
-                    type = type.GetEnumUnderlyingType();
+                    type = type.GetEnumUnderlyingType()!;
                 }
```

Comment:
💭 I believe this could be simplified to just the following without a suppression:

```
type = type.GetEnumUnderlyingTypeOrSelf();
```